managed/services/supervisord/logs.go

```diff
@@ -146,13 +148,23 @@ func (l *Logs) files(ctx context.Context) []fileContent {
 		})
 	}
 
+	b, m, err := readFile("/srv/alertmanager/alertmanager.base.yml")
```

Comment:
Why we hide values only for `alertmanager.base.yml` file, but not for `alertmanager.yml` ?